doc/sfp-cmis/cmis.md

```diff
@@ -0,0 +1,308 @@
+                Bank 0 Implemented
+                Diagnostic Pages Implemented
+                Page 03h Implemented
+        Length SMF(km): 0.0
+        Module State: Ready State
+        Revision Compliance: 4.0
+        Vendor Date Code(YYYY-MM-DD Lot): 2020-10-07
+        Vendor Name: AVAGO
+        Vendor OUI: 00-17-6A
+        Vendor PN: AFCT-93DRPHZ-AZ2
+        Vendor Rev: 01
+        Vendor SN: FD2038FG0FY
+
+admin@sonic:~$ show interfaces transceiver eeprom Ethernet0 --dom
+Ethernet0: SFP EEPROM detected
+        Application Advertisement:
+                1: 400GAUI-8 C2M (Annex 120E) | 400GBASE-DR4 (Cl 124)
+                2: 100GAUI-2 C2M (Annex 135G) | 100G-FR/100GBASE-FR1 (Cl 140)
+        Connector: SN (Mini CS)
+        Identifier: QSFP56-DD
+        Implemented Memory Pages:
+                Bank 0 Implemented
+                Diagnostic Pages Implemented
+                Page 03h Implemented
+        Length SMF(km): 0.0
+        Module State: Ready State
+        Revision Compliance: 4.0
+        Vendor Date Code(YYYY-MM-DD Lot): 2020-10-07
+        Vendor Name: AVAGO
+        Vendor OUI: 00-17-6A
+        Vendor PN: AFCT-93DRPHZ-AZ2
+        Vendor Rev: 01
+        Vendor SN: FD2038FG0FY
+        ChannelMonitorValues:
+                RX1Power: 2.9217dBm
+                RX2Power: 1.9984dBm
+                RX3Power: -40.0000dBm
+                RX4Power: -40.0000dBm
+                TX1Bias: 44.9420mA
+                TX1Power: 2.8587dBm
+                TX2Bias: 47.4440mA
+                TX2Power: 3.0211dBm
+                TX3Bias: 47.4440mA
+                TX3Power: 2.9634dBm
+                TX4Bias: 49.9460mA
+                TX4Power: 2.7395dBm
+        ChannelThresholdValues:
+                RxPowerHighAlarm  : 7.5000dBm
+                RxPowerHighWarning: 4.4999dBm
+                RxPowerLowAlarm   : -10.4001dBm
+                RxPowerLowWarning : -6.4016dBm
+                TxBiasHighAlarm   : 75.0000mA
+                TxBiasHighWarning : 70.0000mA
+                TxBiasLowAlarm    : 10.0000mA
+                TxBiasLowWarning  : 15.0000mA
+                TxPowerHighAlarm  : 6.9999dBm
+                TxPowerHighWarning: 3.9999dBm
+                TxPowerLowAlarm   : -6.4016dBm
+                TxPowerLowWarning : -2.4003dBm
+        ModuleMonitorValues:
+                Temperature: 27.4336C
+                Vcc: 3.2883Volts
+        ModuleThresholdValues:
+                TempHighAlarm  : 75.0000C
+                TempHighWarning: 70.0000C
+                TempLowAlarm   : -5.0000C
+                TempLowWarning : 0.0000C
+                VccHighAlarm   : 3.6300Volts
+                VccHighWarning : 3.4650Volts
+                VccLowAlarm    : 2.9700Volts
+                VccLowWarning  : 3.1350Volts
+```
+
+### 3.3.2 Debug Commands
+
+The **sfputil** will be enhanced for CMIS debugging
+
+# Unit Test
+
+- Verify with QSFPDD static and DOM information with **show interface transceiver eeprom**
```

Comment:
Now there is a set of unit test cases for  xcvrd daemon, would you please also add new cases to cover your new added task?